Scoop/Controller/Ride Posting/PostRideTripDetailsViewController.swift

```diff
@@ -175,6 +176,7 @@ class PostRideTripDetailsViewController: PostRideViewController {
         timePicker.datePickerMode = .time
         timePicker.preferredDatePickerStyle = .wheels
         timePicker.addTarget(self, action: #selector(updateTime), for: .valueChanged)
+        timeTextField.addTarget(self, action: #selector(updateTime), for: .touchDown)
```

Comment:
`touchUpInside`

Reply:
Do any entire sweep of the codebase and make sure you don't use `touchDown` anywhere else